src/styles/main.css

```diff
@@ -0,0 +1,205 @@
+body {
+  margin: 0;
+  display: flex;
+  height: 100vh;
+}
+
+.calendar {
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: calc(100px * 7 + 6px);
+  padding: 0 10px;
+}
+
+.calendar .calendar__day {
+  background-color: #eee;
+  box-sizing: border-box;
+  height: 100px;
+  width: 100px;
+  border: 1px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  color: bla;
```

Comment:
There is a typo in the color property value: 'bla'. It should be corrected to a valid color value, such as 'black' or a hex code like '#000'.